targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_hash.h

```diff
@@ -0,0 +1,76 @@
+/**************************************************************************/
+/* Copyright (c) Samsung Electronics Co., Ltd. All rights reserved.       */
+/*                                                                        */
```

Comment:
Please review the license.  Other files were already updated.

Reply:
@0xc0170. Thanks for the tip and sorry about the inconvenience. I should have checked.

@jh6186 
I grep searched and found out these files without proper SPDX identifiers or inconsistent license setting. Could you please review these files especially regarding legal notice?
```
> targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_hash.c
> targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_hash.h
> targets/TARGET_Samsung/security_subsystem/drivers/sss_common.h
> targets/TARGET_Samsung/security_subsystem/drivers/sss_driver_sha2.c
> targets/TARGET_Samsung/security_subsystem/drivers/sss_driver_sha2.h
> targets/TARGET_Samsung/security_subsystem/drivers/sss_map.h
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/TOOLCHAIN_ARM_STD/sidk_s5js100.sct
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/TOOLCHAIN_ARM_STD/startup_sidk_s5js100.S
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/TOOLCHAIN_GCC_ARM/sidk_s5js100.ld
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/TOOLCHAIN_GCC_ARM/startup_sidk_s5js100.S
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/TOOLCHAIN_IAR/sidk_s5js100.icf
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/TOOLCHAIN_IAR/startup_s5js100.S
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/modem/shmem_save.cpp
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/serial_dummy_api.c
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/serial_pl011_api.c
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/serial_usi_api.c
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/spi_api.c
```
Since they are related with legal setting, I cannot modify them myself.

Reply:
Thank you for the code change. @jh6186.
@0xc0170 
I grepped and verified that every code the Samsung team added now is marked with SPDX Apache 2.0 from https://github.com/ARMmbed/mbed-os/pull/12106/commits/1a9049f5d94dfda115501cfb01350106226ff043.